### ISSUE TITLE
Retrolambda fails after updating Kotlin plugin to 0.11.91.4

### ISSUE THREAD

**Issue by cypressious** [evant/gradle-retrolambda#106]
Since I'm not sure where the problem actually lies, I've also created an issue on the Kotlin tracker: 
https://youtrack.jetbrains.com/issue/KT-7629

After updating from the Gradle Kotlin-Android plugin version 0.11.91.1 to 0.11.91.4 the gradle-retrolambda task started failing with an exception saying that one of my Kotlin classes could not be found. After debugging the build I noticed that on the new version the "build\tmp\kotlin-classes\developmentDebug" folder was missing from the classpath of the retrolambda task.

**Comment by cypressious** [comment]
Hacky workaround for now:

```
project.afterEvaluate {

    def variants = project.android.applicationVariants + project.android.testVariants

    variants.each { var ->
        def task = tasks["compileRetrolambda${var.name.capitalize()}"]
        task.classpath.add(project.files(new File(buildDir, "tmp/kotlin-classes/${var.name}")))
    }

}
```

**Comment by evant** [comment]
This will be fixed in the next release.

**Comment by evant** [comment]
Fixed in 3.1.0
